web/themes/custom/tube_theme/css/style.css

```diff
@@ -7825,6 +7825,12 @@ h3, h4, h5, h6 {
   .node-featured-teaser .featured-author {
     color: #222222;
     font-size: 1.125em; }
+  .node-featured-teaser .field__item {
+    float: none; }
+    .node-featured-teaser .field__item img {
+      display: block;
+      margin-left: auto;
+      margin-right: auto; }
```

Comment:
It's better to commit scss and css files separately. The scss files typically merge cleanly with teammate's PR's, but the compiled css file typically has conflicts.